include/envoy/network/filter.h

```diff
@@ -351,6 +358,8 @@ class FilterChainManager {
   virtual const FilterChain* findFilterChain(const ConnectionSocket& socket) const PURE;
 };
 
+using FilterChainManagerSharedPtr = std::shared_ptr<FilterChainManager>;
```

Comment:
Where is this used and why is it shared? Generally a good idea to approach shared pointers with skepticism, there's often simpler ways of expressing ownership.

Reply:
This is not stripped clearly. Will need in the future PRs. Deleting it in this PR